### DISCUSSION TITLE
Create lighter version of BBN OS

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#361] [Ideas]
It’s quite easy to wrap builds script sections to add support of building a lighter version of BBN OS. 

It will start faster and use less disk space and memory as running.

Question is which apps should be kept. 

OpenCPN but without most plugins
SignalK without most plugins 
AvNav removed
Anything Java removed 
Most radio and SDR stuff removed 
Victron stuff is needed
Pypilot kept 
Mopidy removed 
Still something needed for music. 
VNC server kept
Firewall kept
Browser

email, social networking removed

**Comment by wsturner88** [comment]
keep the ApplePlay Support?   I like the idea of the thin version.
BTW - Anyone try it on a RPi5 Yet?

**Reply by mgrouch** [reply]
Raspotify and shairplay are kept as they are very lightweight. 
yes it was tried on rpi5 (I haven’t tried most recent build but it should be even better)

**Reply by mgrouch** [reply]
Test versions are here

https://cloudsmith.io/~bbn-projects/repos/bbn-repo/packages/

**Reply by AAlbero** [reply]
I like that.
